index.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Coding Theatre</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav class="container">
+      <div class="nav-left container">
+        <i class="fa-solid fa-bars-staggered"></i>
+        <img
+          src="./images/play.png"
+          alt="play"
+          class="play"
+          width="80"
+          height="80"
+        />
+        <span class="logo-name">MyTube</span>
+      </div>
+      <div class="nav-middle container">
+        <div class="search-box container">
+          <input type="text" placeholder="Search" />
+          <i class="fa-solid fa-magnifying-glass"></i>
+        </div>
+        <i class="fa-solid fa-microphone"></i>
+      </div>
+      <div class="nav-right container">
+        <i class="fa-solid fa-square-plus"></i>
+        <i class="fa-solid fa-bell"></i>
+        <img src="./images/me.jpg" alt="me" class="user-icon" />
+      </div>
+    </nav>
```

Comment:
Good job creating the navbar, however, I think it would be better to wrap the <nav> tags inside <header>.  By using the <header> tag to wrap the navbar, it signals to search engines and assistive technologies that this section is the main navigation area for the webpage.

This can improve the accessibility of the website for users who rely on screen readers or other assistive technologies, as well as improve the SEO (search engine optimization) of the website by providing a clear and meaningful context for the content.